crates/ethereum/reth/src/lib.rs

```diff
@@ -21,7 +21,12 @@ pub mod primitives {
 
 /// Re-exported cli types
 #[cfg(feature = "cli")]
-pub use reth_ethereum_cli as cli;
+pub mod cli {
+    #[doc(inline)]
+    pub use reth_cli_util as utl;
```

Comment:
Worried about collisions? Otherwise, we could just
```suggestion
    pub use reth_cli_util::*;
```

Reply:
right, that's better